### ISSUE TITLE
Add a Sketch design template or library

### ISSUE THREAD

**Issue by alexandernanberg** [sitevision/envision#75]
More and more people are moving away from Photoshop so it would be nice to have the design template in Sketch as well. 

Sketch also have a library feature that would simplify things significantly
https://www.sketchapp.com/docs/libraries/
https://medium.com/ux-power-tools/sketch-libraries-how-they-work-and-the-crazy-stuff-you-can-do-with-them-fc10f142ac80

**Comment by melineric** [comment]
A sketch design template is currently in the workings. Not sure of the ETA, but should be done before Christmas.
